packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -32,22 +130,97 @@ export default class AwsS3Multipart extends BasePlugin {
     this.#client = new RequestClient(uppy, opts)
 
     const defaultOptions = {
-      timeout: 30 * 1000,
       limit: 0,
-      retryDelays: [0, 1000, 3000, 5000],
+      retryDelays: [0, 1000, 3000, 5000], // todo this in no longer in use. either implement or remove.
```

Comment:
As for implementing retries and exponential back-off, I would recommend using a library like [ky](https://github.com/sindresorhus/ky) which has this built in. It's a thin layer on top of `fetch`

Reply:
What we need ideally is a shared XHR class that is used by `@uppy/xhr-upload`, `@uppy/aws-s3`, and `@uppy/aws-s3-multipart`. When we get to that we can consider something like `ky`, but I think we should keep it minimal for now

Reply:
Why is it not in use, what happens when chunk fails?

Reply:
```suggestion
      retryDelays: [0, 1000, 3000, 5000],
```